src/error.js

```diff
@@ -35,11 +35,11 @@ export const clearError = () => ({
 });
 
 export const applicationError = (code, error = new Error('application error'), otherData = {}) => (dispatch) => {
-  dispatch(addErrorToReport(error, {
+  dispatch(addErrorToReport(error, JSON.stringify({
```

Comment:
what about doing this in [formatErrorReport](https://github.com/americanexpress/one-app-ducks/blob/master/src/errorReporting.js#L38)